documentation/proton.html

```diff
@@ -118,6 +121,56 @@ <h3 id="sub-databases">Sub-databases</h3>
 
 
 
+<h2 id="transaction-log">Transaction log</h2>
+<p>
+Content nodes have a transaction log to persist mutating operations immediately.
+ToDo: When synced to file system?
+</p><p>
+Operations are written immediately,
+irrespective of <a href="reference/services-content.html#visibility-delay">visibility-delay</a>.
+</p><p>
+Default, proton will
+<a href="reference/services-content.html#flush-on-shutdown">flush the transaction log on shutdown</a>.
```

Comment:
It is not the transaction log that is flushed, but the various components (attribute vectors, memory index, etc) to get faster replay time when restarting.